packages/svg/src/SvgFeatureRenderer/components/SvgFeatureRendering.js

```diff
@@ -90,6 +96,7 @@ export const SvgMouseover = observer(
 
           return (
             <rect
+              style={{ pointerEvents: 'none' }}
```

Comment:
Why is this pointer-events none added

Reply:
This is for the mouseover rects, did they cause trouble?

Reply:
Since the mouseover and selection rects intercepted the click, the onClick on the FeatureGlyph was never getting called.

Reply:
possibly this could be solved with z-index also, just an idea